src/operations.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable max-classes-per-file, no-unused-vars, spaced-comment, no-restricted-syntax*/
+
+class Todo {
+  constructor(description, completed = false, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+class Operations {
+  static addTodo(todos, desc, completed) {
+    const todo = new Todo(desc, completed, todos.length + 1);
+    todos.push(todo);
+  }
+
+  static saveItemsToLocalStorage(items) {
+    localStorage.setItem('todoList', JSON.stringify(items));
+  }
+
+  // static updateIndex(tasks) {
+  //   tasks.forEach((task) => {
+  //     task.index = tasks.indexOf(task) + 1;
+  //   });
+  // }
+
```

Comment:
This part of the code is commented out.  It would be a good idea to remove it if it has no functionality yet. What do you think?

Reply:
Thanks for the review I have implemented the changes